pkg/chunk/aws/dynamodb_table_client.go

```diff
@@ -104,6 +111,21 @@ func (d dynamoTableClient) ListTables(ctx context.Context) ([]string, error) {
 	return table, err
 }
 
+func chunkTagsToDynamoDB(ts chunk.Tags) []*dynamodb.Tag {
+	if ts == nil {
+		return nil
+	}
```

Comment:
Is this redundant?  Looks like `result` will be nil if len(ts) == 0 (or ts is nill)

Reply:
yes; it's cut/pasted from what was there before